app/src/main/res/layout/bottom_sheet_main_menu.xml

```diff
@@ -145,11 +145,11 @@
         android:layout_height="1dp"
         android:background="@color/colorDivider" />
 
-    <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="48dp"
-        android:layout_gravity="center_horizontal"
-        android:gravity="center_vertical">
+    <org.mozilla.focus.widget.EqualDistributeRow xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can't find the spec of this, Let's make sure with Carol.

Reply:
talked with Carol. I'll try to copy the spec from 2.0 for bottom menu
https://drive.google.com/file/d/1ccNdBlAYKekmlgg7f00cKHcJOb9GgRQl/view